src/main/webapp/scripts/account-info.js

```diff
@@ -1,18 +1,30 @@
-
-let auth2;
 let profile;
 const initPromise = init();
 
 
+const API_KEY = 'AIzaSyBD6cw2_iZrPpQlQJswditPC5n3-1I64B0';
```

Comment:
Checking in API key to GitHub is not allowed. From intern host guide: "Cloud Abuse and EIP run a job that revokes service accounts keys found on github due to risk of compromise."

Possible solutions:
- Use git-secrets (recommended) or a .gitignore file to exclude the file containing the service account credentials.(i.e. including /path/to/api_key.json in the .gitignore file)
- Place the file with the API key outside of the Github project and retrieve it by creating an environment variable (i.e. GCP_API_KEY=/path/to/api_key.json).

Reply:
did the .gitignore thing